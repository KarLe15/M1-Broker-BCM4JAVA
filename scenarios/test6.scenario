## operation, component, scenario, uri
CREATE,subscriber,simple_1,subscriber_1
CREATE,subscriber,simple_2,subscriber_2
CREATE,subscriber,simple_3,subscriber_3
CREATE,publisher,no_waiting,publisher_1
CREATE,publisher,no_waiting,publisher_2
CREATE,publisher,no_waiting,publisher_3
CREATE,publisher,no_waiting,publisher_4
