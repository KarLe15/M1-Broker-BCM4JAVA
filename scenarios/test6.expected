{
  "subscriber_1" : 6000,
  "subscriber_2" : 8000,
  "subscriber_3" : 8000
}
